### DISCUSSION TITLE
Income vs expenses Graph on Default Financial Report treats account reconciliations as "spent" money

### DISCUSSION THREAD

**Discussion by pmoberg** [firefly-iii/firefly-iii#4305] [Firefly III] [answered]
I'm running FireFly III v. 5.4.6

If I am on the Dashboard Page and click on the "NET WORTH" card that takes me to the Default Financial Report with the currentStartYear/currentEndYear as parameters. 

The first graph that shows up is the "Income vs expenses" graph. I am a bit surprised to see that this graph seems to treat my account reconciliations as "spent" money and lumps it together with my expenses. 

Is this the correct behavior? 

I have used the reconciliation feature for an account to "correct" the account and didn't think I was spending any money while doing it (I am basically tracking my external stock portfolio account in Firefly by periodically applying a reconciliation to "correct" the amount on the account).

When I add my expenses for the month to my reconciles that matches what the spent bar graph is showing.

**Comment by JC5** [comment, marked as answer]
Reconciliations aren't really meant for this, which is why it may seem like income or expenses. Wouldn't Portfolio Performance be something you can use to track your stock portfolio?

**Reply by pmoberg** [reply]
Thanks for the reply. 

Since the reconciliations are not counted as expenses in other places of the app it has been a great way to track all of my net worth in your great app. It is just this one chart on the Net Worth report that treats them a bit differently. 

That said, this is not a big issue for me since I can just disregard this particular chart and keep using the app. 

I will take a look at Portfolio Performance as well. 

I really enjoy using Firefly III and I appreciate all you do for it and its community.

**Reply by JC5** [reply]
Thanks! I agree with you, reconciliations are a bit unclear at the moment. It is on my list to improve it. Thanks for the compliments!
